packages/wallet-ui/src/components/ui/organism/SideBar/SideBar.view.tsx

```diff
@@ -114,8 +116,18 @@ export const SideBarView = ({ address }: Props) => {
       <AccountLabel>My account</AccountLabel>
       <RowDiv>
         <InfoIcon onClick={() => setInfoModalOpen(true)}>i</InfoIcon>
-        <AccountAddress address={address} starkName={starkName} />
-        <AddIcon onClick={async () => await addNewAccount(chainId)}>+</AddIcon>
+        <AccountSwitchAddress
+          address={address}
```

Comment:
nit:
just to be more explicit, i know it is not standard every where else
but the `AccountSwitchAddress` is taking the current account and the list of accounts as data to display
btw, swall we rename `AccountSwitchAddress` to `AccountSwitchModel`, as it is a model

wdyt?


```suggestion
          currentAccount={address}
```